prog/weave-npc/main.go

```diff
@@ -194,6 +186,17 @@ func createBaseRules(ipt *iptables.IPTables, ips ipset.Interface) error {
 		return err
 	}
 
+	// delete `weave-local-pods` ipset which is no longer used by weave-npc
+	weaveLocalPodExist, err := ipsetExist(ips, npc.LocalIpset)
+	if err != nil {
+		common.Log.Errorf("Failed to destroy ipset '%s'", npc.LocalIpset)
```

Comment:
This message is a little misleading.  It's something like "failed to look for ipset".